File: ruby/lib/uris.rb
class URIS
  class << self
    def agendrix_api
      "https://api.sandbox.agendrix.net"
    end

    def oauth_provider
      "https://sandbox.agendrix.net/oauth"
    end

    def redirect_uri
      "https://localhost:3000/integrations/agendrix/oauth/redirect"
    end
  end
end

File: js/shared/uris.ts
// This file only serves the purpose of the example
// In a real application, those informations as environnement variables
// Since they do not need to stay secret, it would be alright to send them client side in a way or another

const agendrixAPI = "https://api.sandbox.agendrix.net/v1";
const oauthProvider = "https://sandbox.agendrix.net/oauth";
const serverAPI = "http://localhost:4000/api";
const redirectURI = "https://localhost:3000/home";

export { agendrixAPI, oauthProvider, serverAPI, redirectURI };
